ref/general/hardening-intro.adoc

```diff
@@ -22,15 +22,16 @@ Hardening is a general term that is used to describe the measures that are taken
 
 [#operating-system]
 == Operating system
-Operating system intrusions occur when users with local access attempt to cause damage or extract sensitive information. One example of this type of intrusion is malware. If malware is introduced into a system, unauthorized users might be able to access the operating system and its contents.
+Operating system (OS) intrusions occur when users with local access attempt to cause damage or extract sensitive information. One example of this type of intrusion is malware. If malware is introduced into a system, unauthorized users might be able to access the OS and its contents.
 
-To learn more about operating system vulnerabilities and ways to prevent operating system intrusions, go to <link pending>.
+To learn more about OS vulnerabilities and ways to prevent OS intrusions, go to <link pending>.
 
 ==== Best practices
 
-* *Keep Open Liberty up to date*: To reduce operating system vulnerabilities, it's important to keep your version of Open Liberty up to date. Open Liberty fix packs are compatible with later versions, and the zero migration policy means that you can move to a newer fix pack without changing anything in your server configuration. Keeping Open Liberty up to date helps ensure that you have the most recent link:/docs/ref/general/#security-vulnerabilities.html[critical security fixes]. To get the latest version of Open Liberty, go to the link:https://openliberty.io/downloads/[downloads page].
+* *Keep Open Liberty up to date*: To reduce OS vulnerabilities, it's important to keep your version of Open Liberty up to date. Open Liberty fix packs are compatible with later versions, and the zero migration policy means that you can move to a newer fix pack without changing anything in your server configuration. Keeping Open Liberty up to date helps ensure that you have the most recent link:/docs/ref/general/#security-vulnerabilities.html[critical security fixes]. To get the latest version of Open Liberty, go to the link:https://openliberty.io/downloads/[downloads page].
```

Comment:
"Keep Open Liberty up to date". Maybe change to "Stay up to date with Open Liberty"
 
Change,
"To reduce OS vulnerabilities, it's important to keep your version of Open Liberty up to date."
to
"To reduce OS vulnerabilities, it's important to be up to date with your version of Open Liberty ."

Reply:
I updated the first part